### ISSUE TITLE
Accessibility - Image tag doesn't have an alt attribute.

### ISSUE THREAD

**Issue by antleblanc** [Eva-Delpierre/Porfolio#1]
In order to improve a tiny bit the accessibility, I would recommand to add an `alt` attribute to the `<Image>` component.

Source: https://web.dev/image-alt

**Comment by antleblanc** [comment]
Fix here: https://github.com/Eva-Delpierre/Porfolio/commit/3db68055882e4cb2d1c4f6efdac39a1cc0871504#diff-f07226b8c18f4b95ce0b03b0460069cfR15
